ADDING_PROJECTS.md

```diff
@@ -0,0 +1,222 @@
+### YAML Configuration File
+
+Once your project is added to this repository, you **must** create a corresponding YAML configuration file in the `zenml-projects-backend` repository. This file should include all the following attributes:
+
+```yaml
+project_id: your-project-id
+name: Your Project Name
+description: A comprehensive description of what your project does and its key features
+github: https://github.com/zenml-io/zenml-projects/tree/main/your-project-directory
+preview_image: https://github.com/your-org/your-project/raw/main/assets/preview.png
+main_image_link: https://public-flavor-logos.s3.eu-central-1.amazonaws.com/projects/[PROJECT_NUMBER].jpg
+details: |
+  Detailed description of your project including:
+  
+  ### What It Does
+  
+  Explain the main functionality and purpose of your project
+  
+  ### Key Features
+  
+  - Feature 1: Description
+  - Feature 2: Description
+  - Feature 3: Description
+  
+  ### How It Works
+  
+  Step-by-step explanation of how users can use your project
+  
+  ### Architecture
+  
+  Brief overview of the technical architecture
+
+environment_variables:
+  CUSTOM_VAR: 'value'
+  ANOTHER_VAR: 'another_value'
+
+stack:
+  orchestrator: default
+  artifact_store: default
+  # Add other stack components if needed
+
+tags:
+- Domain (e.g., LLMOps, Computer Vision, MLOps)
+- Technology (e.g., PyTorch, TensorFlow, Hugging Face)
+- Use Case (e.g., Classification, Forecasting, NLP)
+
+tools_used:
+- zenml
+- pytorch
+- pandas
+- scikit-learn
+# Add all major tools/libraries used
+
+pipelines:
+- name: Main Pipeline Name
+  description: Description of what this pipeline does
+- name: Secondary Pipeline Name
+  description: Description of secondary pipeline (if applicable)
+
+architecture_diagram: assets/architecture-diagram.png  # Optional
+
+codespace:
+  enabled: true
+  cpu: 4
+  memory: 8
+```
+
+### Required YAML Attributes
+
+| Attribute | Description | Required |
+|-----------|-------------|----------|
+| `project_id` | Unique identifier for your project | ✅ |
+| `name` | Display name of your project | ✅ |
+| `description` | Short description (1-2 sentences) | ✅ |
+| `github` | GitHub URL to your project directory | ✅ |
+| `preview_image` | URL to preview image | ✅ |
+| `main_image_link` | S3 URL to main project image | ✅ |
+| `details` | Detailed markdown description | ✅ |
+| `environment_variables` | Custom environment variables | ❌ |
+| `stack` | ZenML stack configuration | ✅ |
+| `tags` | Categorization tags | ✅ |
+| `tools_used` | List of technologies used | ✅ |
+| `pipelines` | List of pipelines in the project | ✅ |
+| `architecture_diagram` | Path to architecture diagram | ❌ |
+| `codespace` | Codespace configuration | ❌ |
+| `versions` | Version information | ❌ |
+
+## 🖼️ Image Requirements
+
+### Main Project Image
+
+The `main_image_link` should point to an image stored in the S3 bucket:
+```
+https://public-flavor-logos.s3.eu-central-1.amazonaws.com/projects/[PROJECT_NUMBER].jpg
+```
```

Comment:
should add a comment here maybe that this is for zenml internal use or something, since below you say that the image can be in other places as well?